src/app/Fake.DotNet.Testing.NUnit/NUnit3.fs

```diff
@@ -316,7 +316,11 @@ let run (setParams : NUnit3Params -> NUnit3Params) (assemblies : string seq) =
     let assemblies = assemblies |> Seq.toArray
     if Array.isEmpty assemblies then failwith "NUnit: cannot run tests (the assembly list is empty)."
     let tool = parameters.ToolPath
-    let args = buildArgs parameters assemblies
+    let generatedArgs = buildArgs parameters assemblies
+    let path = Path.Combine(Path.GetTempPath(), (sprintf "%s.txt" (Guid.NewGuid().ToString())))
```

Comment:
What about `Path.GetTempFileName()`?

Reply:
I didn't know that was a thing.  Will update (and update a lot of other code too 😂)